Geometry/VeryForwardGeometry/python/dd4hep/v4/geometryRPFromDD_2021_cfi.py

```diff
@@ -1,7 +1,7 @@
 import FWCore.ParameterSet.Config as cms
```

Comment:
I don't quite understand why this Python config is in a versioned directory. I'm not sure if a new `v5` directory is required for this new version of the file. Since the change is so minor, maybe no new version directory is needed.

Reply:
Indeed, I tend to agree that we could keep v4 in this particular case, since this is a matter of removing a spurious line.

Reply:
Hi @diemort 
I think it is a matter of organizing the code. In the same as v2 which is defined as a bug fix, I think we should have v5, instead of edit inside v4. And maybe we just use geometryRPFromDD_2021_cfi.py (outside vN) directly, not just a link to vN.geometryRPFromDD_2021_cfi

Reply:
@srimanob, using geometryRPFromDD_2021_cfi.py outside vN is more prone to errors in case someone forgets to apply the changes that were made in the latest vN and we avoid duplicating files. I'll submit another PR because other non-dd4hep files have to be updated with v5.

Reply:
@diemort , will the PR that you are announcing here imply a creation of a `Geometry/VeryForwardGeometry/python/dd4hep/v5` area, which will contain a link to a (possibly modified)
`Geometry/VeryForwardGeometry/data/dd4hep/v5/geometryRPFromDD_2021.xml`?

If so, do you plan to revert `Geometry/VeryForwardGeometry/python/dd4hep/v4/geometryRPFromDD_2021_cfi.py` to point to  `Geometry/VeryForwardGeometry/data/dd4hep/v4/geometryRPFromDD_2021.xml`?

In that case, it looks to me cleaner and less error prone to introduce a `Geometry/VeryForwardGeometry/python/dd4hep/v5` area since now, and modifying it afterwards, so that v4 will remain unchanged by construction.

At the contrary, if you plan to create a brand new `Geometry/VeryForwardGeometry/data/dd4hep/v6/geometryRPFromDD_2021.xml`, you can link it from the new `Geometry/VeryForwardGeometry/python/dd4hep/v5` area (and `Geometry/VeryForwardGeometry/python/dd4hep/v4` will link the fixed `Geometry/VeryForwardGeometry/data/dd4hep/v5/geometryRPFromDD_2021.xml`, as it is now). In this second case we can merge this PR as it is coded now: just let us know.

Reply:
@perrotta I partially agreed with @srimanob that we should follow the convention and make the changes into a v5, keeping v4 untouched. So I agree with your cleaner suggestion and introduce v5 now. OTOH, I'd still keep the link outside the vN folder pointing to v5, which is the latest geometry description.

Besides, my next PR will update the entry for `v5/RP_Materials.xml` in `Geometry/VeryForwardGeometry/python/v3/geometryRPFromDD_202*.xml` (new v3) and update the links at `Geometry/VeryForwardGeometry/python/geometryRPFromDD_202*.xml`

Reply:
@diemort I was planning to include your changes in this PR, but if you want to submit a separate PR, that would be fine. I may not be able to make the changes until Monday. I was figuring that there is still time for changes since this PR should wait until it can go into 12_3.

Reply:
@cvuosalo no problem, I can make a separate PR as soon as this one is merged.